### PR TITLE
Add Attribution file and extend maketest to check it (#66)

### PR DESCRIPTION
**Pull request by iain-anderson**
Signed-off-by: Iain Anderson <iain@iotechsys.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,7 @@ require (
 	github.com/edgexfoundry/device-sdk-go v1.1.0
 	github.com/edgexfoundry/go-mod-core-contracts v0.1.31
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
+	github.com/golang/snappy v0.0.1 // indirect
```
Comment (anonymouse64) [inline go.mod]: this is strange, any idea why this is pulled in here? I don't see it in the attribution.txt file as well (which probably means we don't use this code in our usage of 3rd party libs, and it's a transitive dependency, but still odd)

Comment (iain-anderson) [reply go.mod]: Looks like it's brought in by the SQL support:
```
$ go mod why github.com/golang/snappy
# github.com/golang/snappy
github.com/edgexfoundry/device-virtual-go/driver
modernc.org/ql/driver
modernc.org/ql
modernc.org/lldb
modernc.org/zappy
modernc.org/zappy.test
github.com/golang/snappy
```

Comment (anonymouse64) [reply go.mod]: Weird, but okay, not an issue just curious

Comment (anonymouse64) [reply go.mod]: thanks for looking into that

**Comment by lenny-intel** [comment]
@iain-anderson , I don't see a `committers` group defined for this repo. 
https://github.com/orgs/edgexfoundry/teams/device-virtual-committers is for the old Java repo.

**Pull request merged**